xbmc/cores/AudioEngine/Sinks/AESinkSNDIO.cpp

```diff
@@ -21,6 +21,7 @@
 #include "system.h"
 #ifdef HAS_SNDIO
 #include "AESinkSNDIO.h"
+#include "cores/AudioEngine/Engines/ActiveAE/ActiveAE.h"
```

Comment:
if this is required, something is wrong with this sink

Reply:
Addressed both review items. Anything else?